### ISSUE TITLE
cant get by check.. keeps saying no class defined

### ISSUE THREAD

**Issue by larrycatania** [ThunderGemios10/TF2#1]
followed everything exactly and when I do /tf2 checkdata mapname ... it says no class defined....and in the file I have all the classes defined , and I got all the correct responses when placing the button...however... I could create a button named goblybook and not get an error and able to place said gobblygook button without an error.

**Comment by ThunderGemios10** [comment]
Something is wrong then... I'll see what i can do. If you would like to help, You can check the code and find bugs and open a pull request.

Cheers.

**Comment by larrycatania** [comment]
Can u look at these, these are after a setup: http://pastebin.com/gxdQg4h9 - data.yml and http://pastebin.com/eTBSnBdX - dustbowl.yml

**Comment by larrycatania** [comment]
Hey in the config.yml there are no classes listed?

**Comment by larrycatania** [comment]
This is the config when generated http://pastebin.com/Zv2vyP9p

**Comment by ThunderGemios10** [comment]
@larrycatania Something is definitely wrong with the configuration generation... Have you found any fixes?

**Comment by larrycatania** [comment]
no, I tried adding the classes back in the config to see if that would work, but it didnt.

**Comment by ThunderGemios10** [comment]
@larrycatania Because. chasesous removed reading from config code.
